### PR TITLE
Add pages for chains

### PR DESCRIPTION
**Pull request by FrederikBolding**
Adds a details page for each chain that can be accessed by clicking the cards on the front page.

This page allows you to connect to additional RPCs for each chain as well as exposes some more information about the the chain to the user.

### CODE REVIEW

**Review by melissamforbs — APPROVED** [review]

**Review by melissamforbs — APPROVED** [review]

**Review by Queenkaz — APPROVED** [review]

**Review by Queenkaz — APPROVED** [review]

**Review by Queenkaz — APPROVED** [review]

**Review by Queenkaz — APPROVED** [review]

**Review by Queenkaz — APPROVED** [review]

**Review by Queenkaz — COMMENTED** [review]
J

**Review by BacajVas — APPROVED** [review]

**Comment by Oliv23ava** [comment]
Aseptado

**Review by gatleas17 — APPROVED** [review]

**Review by gatleas17 — APPROVED** [review]

**Comment by Oliv23ava** [comment]
Si diga . . .

**Review by gatleas17 — APPROVED** [review]

File: src/pages/chain/{Chain.chainId}.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        standard
+      }
+      status
+      faucets
+      slip44
+    }
+  }
+`;
+
+export default ChainPage;
```
Comment (12Gitcat) [inline src/pages/chain/{Chain.chainId}.tsx:120]: I m contact metamask wallet butt my nft is not showing in metamask wallet please help me

**Review by 12Gitcat — APPROVED** [review]

**Review by 12Gitcat — APPROVED** [review]

**Review by ErnestoLara — APPROVED** [review]

**Review by Yedfree — APPROVED** [review]

**Pull request merged**